Content.Server/Power/Components/BatterySelfRechargerComponent.cs

```diff
@@ -6,8 +6,19 @@ namespace Content.Server.Power.Components
     [RegisterComponent]
     public sealed partial class BatterySelfRechargerComponent : Component
     {
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRecharge")] public bool AutoRecharge { get; set; }
+        [DataField]
+        public bool AutoRecharge { get; set; }
 
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargeRate")] public float AutoRechargeRate { get; set; }
+        [DataField]
+        public float AutoRechargeRate { get; set; }
+
+        [DataField]
+        public bool AutoRechargePause { get; set; } = false;
+
+        [DataField]
+        public float AutoRechargePauseTime { get; set; } = 0f;
+
+        [DataField]
+        public float AutoRechargeCooldown { get; set; } = 0f;
     }
 }
```

Comment:
None of these are documented also pretty sure you don't even need most of these?

Reply:
Fixed